docs/README.md

```diff
@@ -1,44 +1,9 @@
 # Documentation
 
-The documentation is available at: [sbi-dev.github.io/sbi](http://sbi-dev.github.io/sbi)
-
-## Building the Documentation
```

Comment:
how are the docs built and versioned with sphinx? Is there an alternative to `mike`, i.e., that enables easy switching between docs per release?

Reply:
readthdocs does it automatically :)